exec/bytecode/func.go

```diff
@@ -179,12 +181,16 @@ func NewFunc(name string, nestDepth uint32) *FuncInfo {
 	return f
 }
 
-func newFuncWith(pkg *Package, name string, nestDepth uint32) *FuncInfo {
+func newFuncWith(pkg *Package, recv *exec.RecvInfo, name string, nestDepth uint32) *FuncInfo {
```

Comment:
exec/bytecode doesn't need to support method.

```go
func (recv RecvType) Foo(args ArgTypes) (ret RetTypes) {
    ...
}
```

is same as:

```go
func RecvType_Foo(recv RecvType, args ArgTypes) (ret RetTypes) {
    ...
}
```

Of course, in Go, the global function name is `(RecvType).Foo`, not `RecvType_Foo`.